disnake/interactions/message.py

```diff
@@ -84,7 +84,7 @@ def values(self) -> Optional[List[str]]:
 
     @cached_slot_property("_cs_component")
     def component(self) -> MessageComponent:
-        """Union[:class:`Button`, :class:`SelectMenu`]: The component the user interacted with"""
+        """Union[:class:`Button`, :class:`BaseSelectMenu`]: The component the user interacted with"""
```

Comment:
versionchanged

Reply:
This is a documentation-only change, the `MessageComponent` return type is more concrete and unchanged compared to before. I didn't really feel like listing all select types here (once the remaining 4 get added).